docs/guides/configuration.md

```
@@ -0,0 +1,143 @@
+    
+    # Store configuration
+    await ConfigurationRepository.define(config)
+    
+    # Remove configuration
+    await ConfigurationRepository.remove(DatabaseConfig)
+    
+    # List available configurations
+    available = await ConfigurationRepository.available_configurations()
+```
+
+That's it! The configuration system is designed to be simple and integrate seamlessly with Haiway's context and state management.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)